components/automate-chef-io/content/docs/ldap.md

```diff
@@ -1,82 +1,308 @@
 +++
```

Comment:
The configuration.md had this outline:
```
#### Authentication via Existing Identity Management Systems
##### Microsoft Active Directory
##### LDAP
##### SAML
```
The MSAD and LDAP excerpted parts in this file now has expanded headings (👍 ):
```
## Microsoft Active Directory
## Changing Chef Automate Configuration
### Minimal MSAD Configuration
#### Minimal MSAD config.toml
### Full MSAD Configuration
#### Example Full MSAD config.toml
### Extended LDAP Settings
#### Example Extended LDAP config.toml
### Sign In with LDAP
#### Authorization with LDAP
```
But what I do not see there is that the original LDAP and MSAD sections were siblings. I think the LDAP section starts at `Extended LDAP Settings` but that should be a level 2 heading, then, not level 3.

Reply:
I agree. How about:

## Microsoft Active Directory (MSAD)
### Minimal MSAD Configuration
#### Minimal MSAD config.toml
### Full MSAD Configuration
#### Example Full MSAD config.toml
## Lightweight Directory Access Protocol (LDAP)
### LDAP Settings
#### Example LDAP config.toml
### Sign In with LDAP
#### Authorization with LDAP
## Changing Chef Automate Configuration

Reply:
LDAP and MSAD are siblings per the statement in the Overview section - "This is documentation for configuring Chef Automate’s Lightweight Directory Application Protocol (LDAP) and Microsoft Active Directory (MSAD) integrations."